src/remotestorage.js

```diff
@@ -341,13 +341,20 @@
         this.remote.configure(info);
         if (! this.remote.connected) {
           if (info.authURL) {
-            this.authorize(info.authURL);
+            if (typeof token === 'undefined') {
```

Comment:
I think it would make more sense to swap the if and else parts - check if the token is valid (a string?)
and otherwise continue with the authorization sequence.